content/docs/v2.3/libraries-and-tools.md

```diff
@@ -19,7 +19,8 @@ title: Libraries and Tools
 
 **Go libraries**
 
-- [etcd/client](https://github.com/etcd-io/etcd/blob/master/client) - the officially maintained Go client
+- [etcd/client/v3](https://github.com/etcd-io/etcd/blob/master/client/v3) - the officially maintained Go client for v3
```

Comment:
Hmm, ... in principle the v2 docs are in the state they were back then. So it might not make sense to add a link to the v3 client.

Reply:
Comments about this?

Reply:
I agree, updated.